python/lsst/ap/association/filterDiaSourceCatalog.py

```diff
@@ -188,6 +194,21 @@ def run(self, diaSourceCat, diffImVisitInfo):
             diaSourceCat = diaSourceCat[~sky_source_column].copy(deep=True)
             self.log.info(f"Filtered {num_sky_sources} sky sources.")
 
+        if self.config.doRemoveCrCenter:
+            crCenter_mask = diaSourceCat["pixelFlags_crCenter"]
```

Comment:
We may end up wanting to filter multiple flags, so please add a configurable `badFlagList` (or a better name!) and iterate through the flags rather than having hard-coded configs for each.

Reply:
I was actually going to ask about that in my 1 on 1 with Eric tomorrow! I agree with you, I think that's a much more flexible option. :)